doc/changes.rst

```diff
@@ -387,6 +387,9 @@ Changes to individual options
 ``disable_excludes``
   * To disable all configuration file excludes, the ``*`` glob character is used now instead of the ``all`` to unify the behavior with query objects on the API.
 
+``keepcache``
+  * The behavior has been slightly modified, see the :ref:`Caching <caching_packages_ref-label>` man page for more information.
```

Comment:
I would find it more helpful if we directly said what is the change.

Reply:
Sure, me too. Though I am not sure I'd come up with a concise and short description. Also, I am not entirely sure about the behavior of all use cases in dnf4, but I can try to examine...